Db/src/main/java/org/gusdb/fgputil/db/runner/SQLRunner.java

```diff
@@ -195,6 +201,29 @@ public SQLRunner(Connection conn, String sql, String sqlName) {
     _sqlName = sqlName;
   }
 
+  /**
+   * Set a global connection timeout that gets used for all SQLRunner instances if no override is set. Set to null
+   * if no global default timeout is desired.
+   *
+   * @param timeout Default timeout value.
+   */
+  public static void setGlobalDefaultConnectionTimeout(Duration timeout) {
```

Comment:
I feel like we only need one of [ global, default ] here in the name.  I think I prefer default.